jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -81,7 +81,7 @@
             type="number"
             v-model.number="trace_pixel"
             :rules="[() => trace_pixel!=='' || 'This field is required']"
-            :hint="trace_type_selected === 'Flat' ? 'Pixel number/column for the trace.' : 'Pixel number/column guess'"
+            :hint="trace_type_selected === 'Flat' ? 'Pixel row for flat trace.' : 'Pixel row initial guess for auto trace.'"
```

Comment:
we'll just need to remember to update this "auto trace" if we ever add more options, but I think this is best for now.